frontend/src/tests/ForestClientCard.spec.ts

```diff
@@ -0,0 +1,138 @@
+        forest_client_number: TEST_INACTIVE_FOREST_CLIENT_NUMBER,
+        status: {
+            status_code: STATUS_CODE_INACTIVE,
+            description: STATUS_DESCRIPTION_INACTIVE,
+        },
+    }
+];
+
+const testMultipleInactiveClient: FamForestClient[] = [
+    {
+        client_name: `${TEST_FOREST_CLIENT_NAME}_${TEST_INACTIVE_FOREST_CLIENT_NUMBER}`,
+        forest_client_number: TEST_SUCCESS_FOREST_CLIENT_NUMBER,
+        status: {
+            status_code: STATUS_CODE_INACTIVE,
+            description: STATUS_DESCRIPTION_INACTIVE,
+        },
+    },
+    {
+        client_name: `${TEST_FOREST_CLIENT_NAME}_${TEST_INACTIVE_FOREST_CLIENT_NUMBER_2}`,
+        forest_client_number: TEST_INACTIVE_FOREST_CLIENT_NUMBER_2,
+        status: {
+            status_code: STATUS_CODE_INACTIVE,
+            description: STATUS_DESCRIPTION_INACTIVE,
+        },
+    },
+    {
+        client_name: `${TEST_FOREST_CLIENT_NAME}_${TEST_INACTIVE_FOREST_CLIENT_NUMBER_3}`,
+        forest_client_number: TEST_INACTIVE_FOREST_CLIENT_NUMBER_3,
+        status: {
+            status_code: STATUS_CODE_INACTIVE,
+            description: STATUS_DESCRIPTION_INACTIVE,
+        },
+    }
+];
+
+describe('ForestClientCard', () => {
+
+    let wrapper: VueWrapper;
+
+    beforeEach(async () => {
+        wrapper = mount(ForestClientCard);
+    });
+
+    afterEach(() => {
+        wrapper.unmount();
+    });
+
+    it('should show inactive for forest client card in error case', async () => {
+        // Error case
+        await wrapper.setProps({ forestClientData: testInactiveClient });
+        expect(wrapper.get('#forest-client-status').element.textContent).toBe(STATUS_DESCRIPTION_INACTIVE);
+    });
+
+    it('should show active forest client card in successful case', async () => {
+        // Success case
+        await wrapper.setProps({ forestClientData: testActiveClient });
+        expect(wrapper.get('#forest-client-status').element.textContent).toBe(STATUS_DESCRIPTION_ACTIVE);
+
+    });
+
+    it('should show multiple active forest client card in successful case', async () => {
+        // Multiple success case
+        await wrapper.setProps({ forestClientData: testMultipleActiveClient });
+        const clientList = wrapper.findAll('#forest-client-status') as DOMWrapper<Element>[];
+        for (let index in clientList) {
+            expect(clientList[index].element.textContent).toBe(STATUS_DESCRIPTION_ACTIVE);
+        }
+    });
+
+    it('should show multiple inactive forest client card in error case', async () => {
+        // Multiple success case
+        await wrapper.setProps({ forestClientData: testMultipleInactiveClient });
+        const clientList = wrapper.findAll('#forest-client-status') as DOMWrapper<Element>[];
+        for (let index in clientList) {
+            expect(clientList[index].element.textContent).toBe(STATUS_DESCRIPTION_INACTIVE);
+        }
+    });
```

Comment:
Hi @NickSaglioni , not sure if this case is already done somewhere else (or other test file for similar test case). I think now you can add a new test that "mix success, failure, inactive together in the forestClientData list" and verify if client number vs client status are what you expected.
Thanks Nick!

Reply:
I think I saw somewhere you had "mix" case.
Resolve this issue. Thanks.